alchemy/src/planetscale/api/client.gen.ts

```diff
@@ -0,0 +1,2882 @@
+// This file is auto-generated by scripts/planetscale/generate.ts. Do not edit manually.
```

Comment:
@Mkassabov The actual generated file is `.gen.ts` and has the banner.

Removed it from the template to make it clear that the template isn't actually generated.

I guess I find it a little weird to put `.gen.ts` on a file that isn't actually generated, but I don't have too strong of an opinion.